inference-engine/tests/functional/plugin/shared/src/single_layer_tests/eltwise.cpp

```diff
@@ -7,149 +7,130 @@
 
 #include <map>
 
-#include "common_test_utils/common_layers_params.hpp"
-#include "common_test_utils/common_utils.hpp"
-#include "common_test_utils/test_common.hpp"
-#include "common_test_utils/test_constants.hpp"
-#include "common_test_utils/xml_net_builder/ir_net.hpp"
-#include "common_test_utils/xml_net_builder/xml_filler.hpp"
 #include "functional_test_utils/layer_test_utils.hpp"
 #include "ngraph_functions/builders.hpp"
 #include "ngraph_functions/utils/ngraph_helpers.hpp"
-#include "ie_core.hpp"
+
 #include "single_layer_tests/eltwise.hpp"
 
-using namespace EltwiseTestNamespace;
+namespace LayerTestsDefinitions {
+const std::string EltwiseParams::InputLayerType_to_string(InputLayerType lt) {
+    switch (lt) {
+        case InputLayerType::CONSTANT:
+            return "CONSTANT";
+        case InputLayerType::PARAMETER:
+            return "PARAMETER";
+        default:
+            return "NOT_SUPPORTED_INPUT_LAYER_TYPE";
+    }
+}
 
-std::string EltwiseLayerTest::getTestCaseName(testing::TestParamInfo<eltwiseLayerTestParamsSet> obj) {
-    EltwiseOpType op;
-    ParameterInputIdx primary_input_idx;
-    InputLayerType secondary_input_type;
-    InferenceEngine::Precision prec;
-    InferenceEngine::SizeVector vec;
-    LayerTestsUtils::TargetDevice dev;
+const std::string EltwiseParams::EltwiseOpType_to_string(EltwiseOpType eOp) {
+    switch (eOp) {
+        case EltwiseOpType::ADD:
+            return "Sum";
+        case EltwiseOpType::MULTIPLY:
+            return "Prod";
+        case EltwiseOpType::SUBSTRACT:
+            return "Sub";
+        default:
+            return "NOT_SUPPORTED_ELTWISE_OPERATION";
+    }
+}
+
+const std::string EltwiseParams::OpType_to_string(OpType op) {
+    switch (op) {
+        case OpType::SCALAR:
+            return "SCALAR";
+        case OpType::VECTOR:
+            return "VECTOR";
+        default:
+            return "NOT_SUPPORTED_ADDITION_TYPE";
```

Comment:
For either Substraction or Multiplication it would be strange to have NOT_SUPPORTED_ADDITION_TYPE returned